sklearn/tree/_tree.pyx

```diff
@@ -63,7 +64,7 @@ TREE_UNDEFINED = -2
 cdef SIZE_t _TREE_LEAF = TREE_LEAF
 cdef SIZE_t _TREE_UNDEFINED = TREE_UNDEFINED
 cdef SIZE_t INITIAL_STACK_SIZE = 10
-cdef DTYPE_t MIN_IMPURITY_SPLIT = 1e-7
+cdef DTYPE_t LEAF_MIN_IMPURITY = 1e-7
```

Comment:
In fact, couldnt we get rid of this constant and set `min_impurity_split=1e-7` by default? It shouldnt change the present behaviour, right?